src/lib/gameLogic/GameLobby.js

```diff
@@ -221,6 +224,24 @@ class GameLobby {
     return Math.floor((this.drawTimer / this.timerLength) * 100);
   }
 
+  updateMessage(message) {
+    this.messages.push(message);
+    console.log('message:', message);
+    // console.log("MESSAGE FROM GAMELOBBY : ", this.messages);
+    let chatMessage = '';
+    if (message.evaluate) {
+      chatMessage = `${message.userName} guessed the word.`;
+    } else {
```

Comment:
might want to think about if in the future you would want to separate username from message for styling and process that on the front end but think this makes sense for now! also not sure if you want the message to appear differently if is correct or not ie green text, and therefore would want a success boolean or some other property